.github/workflows/commands.yml

```diff
@@ -1,14 +0,0 @@
-name: commands
-on:
-  issue_comment:
-    types: [created]
-
-jobs:
-  retest:
-    name: Retest
```

Comment:
FYI this means that `/retest` won't retest failed GitHub Actions jobs.

Where could I look at how Repokitteh is setup so maybe I could teach it to do that?

Reply:
I'm fine adding it back, I just was trying to start with CI workflows that verified we didn't break the build :-)
repokitteh files for Envoy are here:  ci/repokitteh/modules

Reply:
related #24156